tests/bgp/conftest.py

```diff
@@ -359,21 +358,21 @@ def _setup_interfaces_t1_or_t2(mg_facts, peer_count):
                 for pt in mg_facts["minigraph_portchannel_interfaces"]:
                     if _is_ipv4_address(pt["addr"]):
                         pt_members = mg_facts["minigraph_portchannels"][pt["attachto"]]["members"]
+                        pc_asic_idx = duthost.get_asic_index_for_portchannel(pt["attachto"])
                         # Only use LAG with 1 member for bgpmon session between PTF,
                         # It's because exabgp on PTF is bind to single interface
                         if len(pt_members) == 1:
                             # If first time, we record the asic index
-                            if not ipv4_lag_interfaces:
+                            if not ipv4_interfaces and not ipv4_lag_interfaces:
```

Comment:
why we need this change?

Reply:
discussed offline with @ysmanman. This change is correct and need for disagg T2 tests